<ui version="4.0" >
 <class>RelativeParameterClass</class>
 <widget class="QWidget" name="RelativeParameterClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>566</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <family>MS Shell Dlg 2</family>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle" >
   <string>RelativeParameter</string>
  </property>
  <widget class="QPushButton" name="staBtn" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>330</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>统计>></string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>150</y>
     <width>711</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>551</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>功能提示：根据某个参数的范围或者百分位数查询其他参数的统计信息</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>140</y>
     <width>201</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="title" >
    <string>选择待查参数</string>
   </property>
   <widget class="QComboBox" name="tableComboBox" >
    <property name="geometry" >
     <rect>
      <x>79</x>
      <y>31</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>11</x>
      <y>31</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text" >
     <string>选择分组：</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>70</y>
      <width>181</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>40</y>
     <width>851</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="title" >
    <string>选择限制条件</string>
   </property>
   <widget class="QComboBox" name="groupComboBox" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>31</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="parameterComboBox" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>60</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>Felix Titling</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>13</x>
      <y>31</y>
      <width>65</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text" >
     <string>选择分组：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>13</x>
      <y>60</y>
      <width>65</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text" >
     <string>选择参数：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typeComboBox" >
    <property name="geometry" >
     <rect>
      <x>380</x>
      <y>50</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>510</x>
      <y>30</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="label_6" >
       <property name="font" >
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>参数范围：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="minLineEdit" >
       <property name="font" >
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>----</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="maxLineEdit" >
       <property name="font" >
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>283</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text" >
     <string>选择查询类型：</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>510</x>
      <y>70</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="label_7" >
       <property name="font" >
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>百 分 位：P</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="centileLineEdit" >
       <property name="font" >
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
